src/components/NavBar/SearchBar.tsx

```diff
@@ -105,78 +121,116 @@ export const SearchBar = () => {
     hasInput: debouncedSearchValue && debouncedSearchValue.length > 0,
     ...trace,
   }
+  const placeholderText =
+    phase1Flag === NftVariant.Enabled
+      ? isMobileOrTablet
```

Comment:
nit: Nested tertiary statements can be pretty confusing. My preference is to extract the logic out into a switch statement

Reply:
you should also have this text be wrapped in useMemo with conditions on phase1Flag and isMobileOrTablet to prevent extraneous rerendering

Reply:
i don't think the memo would be more efficient in this case fwiw

Reply:
ah - wrapped in anyway in the latest push